### PR TITLE
Update aws-kubernetes for SEO.

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu <patrickluo@yunify.com>
/assign @FeynmanZhou

### CODE REVIEW

**Comment by FeynmanZhou** [comment]
@Patrick-LuoYu Please consider adding these keywords in this blog:
image

**Comment by Patrick-LuoYu** [comment]
The article is further updated. Please take a look. Thanks! @FeynmanZhou

File: content/en/blogs/aws-kubernetes.md [hunk 1]
```diff
@@ -131,13 +133,13 @@ sudo systemctl restart sshd
...
 
-Download KubeKey from the [Github Realese Page](https://github.com/kubesphere/kubekey/releases) or run the following command:
+Download KubeKey from the [Github Release Page](https://github.com/kubesphere/kubekey/releases) or run the following command:
 
 ```
 curl -sfL https://get-kk.kubesphere.io | VERSION=v1.1.1 sh -
 ```
 
-## Use KubeyKey to Deploy a Cluster
+## Use KubeKey to Deploy a Cluster
```
Comment (FeynmanZhou) [inline content/en/blogs/aws-kubernetes.md:142]: Can we use `Deploy a Kubernetes Cluster`?

**Comment by Patrick-LuoYu** [comment]
Further improved. @FeynmanZhou

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**